packages/caip/src/adapters/coingecko/index.ts

```diff
@@ -44,7 +43,7 @@ export const coingeckoToAssetIds = (id: CoinGeckoId): AssetId[] =>
   generatedCoingeckoToAssetIdsMap[id]
 
 export const assetIdToCoingecko = (assetId: AssetId): CoinGeckoId | undefined =>
-  generatedAssetIdToCoingeckoMap[toLower(assetId)]
```

Comment:
I didn't see any problems with this, assetIdToCoingecko is consumed 3 time and seems ok as it is without lower case

It was breaking for the solana part